dart-impl/mpi/src/dart_communication.c

```diff
@@ -667,7 +658,13 @@ dart_ret_t dart_put_handle(
                    "(from local allocation) "
                    "target_unit:%d offset:%"PRIu64"",
                    nelem, dtype, target_unitid_abs, offset);
```

Comment:
Do you mean the `%"PRIu64"` strings? They are macros used to correctly print the specified types (PRInt unsigned64).